keyboards/handwired/traveller/keymaps/default/keymap.c

```diff
@@ -145,26 +145,14 @@ Right hand nav keys work pretty well chorded with the Right hand Hi Key
 
 [_TRNS] = {
   {KC_TRNS,  KC_TRNS,   KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_NO,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_TRNS},
-  {KC_TRNS,  KC_TRNS,   KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_NO,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, 	KC_TRNS},
-  {KC_TRNS,  KC_TRNS,   KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS, 	KC_TRNS},
+  {KC_TRNS,  KC_TRNS,   KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_NO,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_TRNS},
+  {KC_TRNS,  KC_TRNS,   KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_TRNS,  KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,   KC_TRNS},
```

Comment:
```suggestion
  {_______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______},
```
Maybe replace occurrences of `KC_TRNS` with `_______` while you're at it. Each column should be 9 characters wide (7 for keycode, 1 for comma, 1 for space) where possible. Where not possible, i.e. where the keycode is too long (e.g. `TG(_FKEYS)`, either use a shorter name (`TG(_FN)`) or add extra spaces to that _column_ only.

Reply:
I think that may be a little out of scope for this PR. IMO if people want to use `KC_TRNS` instead of `_______`, that's their choice, and to avoid stepping on more toes than I have already, I'm going to leave it alone :)

That said, an argument could be made to remove `KC_TRNS` and `KC_NO` altogether, considering they're kind of extra special keycodes separate from even the ones defined in `quantum_keycodes.h`.

Reply:
Yeah, you're probably right.